cerebras_tools_proxy.py

```
@@ -0,0 +1,562 @@
+    except Exception as e:
+        print(f"❌ Error processing request: {e}")
+        traceback.print_exc()
+        raise HTTPException(status_code=500, detail=str(e))
+
+if __name__ == "__main__":
+    print("\n🧠 Tool-Enabled Cerebras Proxy Server")
+    print("=" * 40)
+    print(f"🌐 Port: {API_PORT}")
+    print(f"🤖 Cerebras: {CEREBRAS_BASE_URL}")
+    print(f"📱 Model: {DEFAULT_MODEL}")
+    print(f"🔧 Tools: bash, str_replace_editor, write_file")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)